onnx_tf/handlers/backend/resize.py

```diff
@@ -136,13 +173,18 @@ def version_10(cls, node, **kwargs):
     else:
       mode = tf.image.ResizeMethod.NEAREST_NEIGHBOR
 
+    # process tf.image.resize unsupported datatype for x
+    x = tf.cast(x, cls.x_cast_map[x_dtype]) if x_dtype in cls.x_cast_map else x
```

Comment:
@winnietsang I happened to stumble upon this line when trying to convert [FasterRCNN-10.onnx](https://github.com/onnx/models/blob/master/vision/object_detection_segmentation/faster-rcnn/model/FasterRCNN-10.onnx). I still had your patch applied in my local install.

```
  File "/Users/pluradj/src/github/pluradj/onnx-dev/tf-2/onnx-tensorflow/onnx_tf/handlers/backend/resize.py", line 177, in version_10
    x = tf.cast(x, cls.x_cast_map[x_dtype]) if x_dtype in cls.x_cast_map else x
NameError: name 'x_dtype' is not defined
```

The conversion still fails after fixing by adding `x_dtype = x.dtype` earlier in this method because `RoiAlign` is not yet implemented and outside scope of this PR ;)

Reply:
Thanks @pluradj good catch.

Reply:
LGTM